localflavor/au/forms.py

```diff
@@ -63,3 +65,22 @@ class AUStateSelect(Select):
     """
     def __init__(self, attrs=None):
         super(AUStateSelect, self).__init__(attrs, choices=STATE_CHOICES)
+
+
+class AUBusinessNumberField(CharField):
+    """
+    A form field that validates input as an Australian Business Number (ABN)
+
+    .. versionadded:: 1.3
+    """
+
+    default_validators = [AUBusinessNumberFieldValidator()]
+
+    def clean(self, value):
+        """
+        Return a cleaned ABN.
+        """
+        value = super(AUBusinessNumberField, self).clean(value)\
```

Comment:
There should be a space before the '\'. 

You don't actually need to put the '.replace(...)' on a separate line because we're using 120 character lines in this project. That said, we're not enforcing a specific style yet so you can keep the shorter lines if that's your preference.